src/lib/transaction_snark/transaction_snark.ml

```diff
@@ -99,15 +99,48 @@ module Statement = struct
     {source; target; fee_excess; proof_type; supply_increase}
 end
 
-type t =
+module Stable = struct
+  module V1 = struct
+    module T = struct
+      let version = 1
+
+      type t =
+        { source: Frozen_ledger_hash.Stable.V1.t
+        ; target: Frozen_ledger_hash.Stable.V1.t (* TODO : version *)
```

Comment:
remove TODO?

Reply:
With PR #1965, I can use the version. Made the change.